<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>QuickModAddFileDialog</class>
 <widget class="QDialog" name="QuickModAddFileDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>450</width>
    <height>284</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>450</width>
    <height>153</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Add QuickMod file</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QFormLayout" name="formLayout">
     <property name="fieldGrowthPolicy">
      <enum>QFormLayout::AllNonFixedFieldsGrow</enum>
     </property>
     <item row="0" column="0">
      <widget class="QRadioButton" name="fileButton">
       <property name="text">
        <string>File:</string>
       </property>
       <property name="checked">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <layout class="QHBoxLayout" name="horizontalLayout_3">
       <item>
        <widget class="QLineEdit" name="fileEdit"/>
       </item>
       <item>
        <widget class="QPushButton" name="browseButton">
         <property name="text">
          <string>Browse</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item row="1" column="0">
      <widget class="QRadioButton" name="urlButton">
       <property name="text">
        <string>URL:</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QLineEdit" name="urlEdit">
       <property name="enabled">
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item row="2" column="0" colspan="2">
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-weight:600;&quot;&gt;Protip:&lt;/span&gt; You can also drag and drop files and URLs to anywhere onto MultiMC&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
       </property>
       <property name="wordWrap">
        <bool>true</bool>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="addButton">
       <property name="text">
        <string>Add</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="cancelButton">
       <property name="text">
        <string>Cancel</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>fileButton</sender>
   <signal>toggled(bool)</signal>
   <receiver>fileEdit</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>40</x>
     <y>21</y>
    </hint>
    <hint type="destinationlabel">
     <x>108</x>
     <y>19</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>urlButton</sender>
   <signal>toggled(bool)</signal>
   <receiver>urlEdit</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>43</x>
     <y>50</y>
    </hint>
    <hint type="destinationlabel">
     <x>114</x>
     <y>52</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>fileButton</sender>
   <signal>toggled(bool)</signal>
   <receiver>browseButton</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>42</x>
     <y>23</y>
    </hint>
    <hint type="destinationlabel">
     <x>394</x>
     <y>21</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>urlEdit</sender>
   <signal>returnPressed()</signal>
   <receiver>addButton</receiver>
   <slot>click()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>184</x>
     <y>53</y>
    </hint>
    <hint type="destinationlabel">
     <x>306</x>
     <y>263</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>fileEdit</sender>
   <signal>returnPressed()</signal>
   <receiver>addButton</receiver>
   <slot>click()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>258</x>
     <y>26</y>
    </hint>
    <hint type="destinationlabel">
     <x>301</x>
     <y>263</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
